src/components/Summary/InterviewSummaryModal/index.js

```diff
@@ -1,15 +1,33 @@
 import React from 'react';
-import { Typography, Modal, Row, Col, Card, Rate, Empty } from 'antd';
+import {
+  Typography,
+  Modal,
+  Row,
+  Col,
+  Card,
+  Rate,
+  Empty,
+  Result,
+  Button,
+  Form,
+  message,
+  Input,
+} from 'antd';
 import PropTypes from 'prop-types';
-import { Connect } from 'aws-amplify-react';
-import { graphqlOperation } from 'aws-amplify';
+import { Connect, propStyle } from 'aws-amplify-react';
+import { API, graphqlOperation } from 'aws-amplify';
 import PageEmpty from 'components/PageEmpty';
 import PageSpin from 'components/PageSpin';
 import QuestionComment from 'components/Summary/QuestionComment';
 import { onCreateResult } from 'graphql/subscriptions';
-import { getTest } from './queries';
+import { getTest, listTest, getTest2, getallsnapcomments } from './queries';
+import createComment from 'utils/comment/comment';
 
+//import _ from 'lodash';
+//This is react stateless function component but the hook function is blocked
 const toInterviewResult = data => {
+  const non_null = data.users.items.filter(x => x != null);
+  data.users.items = non_null;
```

Comment:
we usually don't mutate the parameter directly.
In your case, you wanna filter out the null data. You can try this below:
```javascript
// Use filter to keep valid data only, and map it to generate interviewers data
const interviewers = data.users.items
.filter(x => x)
.map(v => v.user);
```
In the above method, it will not modify `data` parameter, we should always keep it pristine.